<launch>
<!-- Launch file for Livox AVIA LiDAR -->

	<arg name="rviz" default="true" />

	<!-- <arg name="depth_width"       value="640"/>
	<arg name="depth_height"      value="480"/>
	<arg name="depth_fps"         value="30"/>
	<arg name="color_width"       value="640"/>
	<arg name="color_height"      value="480"/>
	<arg name="color_fps"         value="30"/> -->

	<rosparam command="load" file="$(find eq_lio)/config/avia.yaml" />

	<param name="feature_extract_enable" type="bool" value="0"/>
	<!-- 100HZ的bag  point_filter_num建议设置为1;   10HZ的bag建议设置为2或3 -->
	<param name="point_filter_num" type="int" value="2"/>               
	<param name="max_iteration" type="int" value="3" />
	<param name="filter_size_surf" type="double" value="0.5" />
	<param name="filter_size_map" type="double" value="0.5" />
	<param name="cube_side_length" type="double" value="1000" />
    <node pkg="eq_lio" type="fastlio_mapping" name="laserMapping" output="screen" /> 

	<group if="$(arg rviz)">
	<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find eq_lio)/rviz_cfg/loam_livox.rviz" />
	</group>

</launch>
